izanami-frontend/src/pages/users.tsx

```diff
@@ -82,18 +83,32 @@ export function Users() {
       },
     }
   );
-
-  const inviteUserMutation = useMutation(
-    (data: { email: string; admin: boolean; rights: TRights }) =>
-      createInvitation(data.email, data.admin, data.rights)
-  );
+  const inviteUserMutation = useMutation<
+    { invitationUrl?: string } | null,
```

Comment:
This whole type is very confusing, invitationUrl is listed but never actually used, Error is too generic. I woulc prefer to just type `data` as it was done before and let Typescript infer the rest.